app/models/nic/base.rb

```diff
@@ -148,6 +151,13 @@ def host_managed?
 
     protected
 
+    def compute_resource_requires_fields
+      if host.compute_resource.present?
+        host.errors.add(:compute_profile, _("you must specify either compute attributes or a compute profile"))
```

Comment:
Best not to use the second person here, just use a fragment so the full sentence makes more sense, e.g. `:compute_attributes, _("is required, or use a compute profile")` (and changed the target since the error is about attributes, a profile is a helper.)